pkg/vsphere/toolbox/vix_property.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package toolbox
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+)
+
+const (
+	_ = iota // ANY type not supported
+	vixPropertyTypeInt32
+	vixPropertyTypeString
+	vixPropertyTypeBool
+	_ // HANDLE type not supported
+	vixPropertyTypeInt64
+	vixPropertyTypeBlob
+)
+
+const (
```

Comment:
All of the [Vv]ix constants are from vix.h, I was thinking about moving them all to a vix.go.  Maybe even generate vix.go from vix.h, but there's a bunch that we don't need.